helm/chart-operator/values.yaml

```diff
@@ -1,3 +1,8 @@
+# For CAPI clusters this will be set to true. So charts for CNI apps can be installed.
+chartOperator:
```

Comment:
The chartOperator here is a bit redundant. But it's so it's clear in the cluster values configmap that will be managed by cluster-apps-operator.